homeassistant/components/ezviz/config_flow.py

```diff
@@ -369,7 +374,9 @@ async def async_step_reauth_confirm(
 
         data_schema = vol.Schema(
             {
-                vol.Required(CONF_USERNAME, default=entry.title): vol.In([entry.title]),
+                vol.Required(CONF_USERNAME, default=entry.unique_id): vol.In(
```

Comment:
Should it default to the current username stored in the entry data?

Reply:
The username for the main account is not stored in data. I have added it.

The entry unique id can't change if I'm not mistaken? So this should be ok? I can add code to _init_ to fix this if that would be better?